helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -639,9 +644,17 @@ exporters:
   {{- end }}
   {{- if (eq (include "splunk-otel-collector.platformLogsEnabled" .) "true") }}
   {{- include "splunk-otel-collector.splunkPlatformLogsExporter" . | nindent 2 }}
+  {{- include "splunk-otel-collector.splunkPlatformSendingQueue" . | nindent 4 }}
+      {{- if .Values.splunkPlatform.sendingQueue.persistentQueueEnabled.logs }}
+      storage: file_storage/persistent_queue
+      {{- end }}
```

Comment:
Why don't keep this inside the `splunk-otel-collector.splunkPlatformLogsExporter` template? This became a bit hard to read

Reply:
This is because `splunk-otel-collector.splunkPlatformLogsExporter` is used in all three agent, cluster receiver and gateway. The problem is that currently we decided to add persistent queue only in agent, because using it in others caused some errors, that no one I contacted was able to resolve.

Reply:
> The problem is that currently we decided to add persistent queue only in agent, because using it in others caused some errors, that no one I contacted was able to resolve.

I don't think it's a valid reason to introduce the inconsistency

Reply:
I created a new issue now #800